data/net/minecraft/world/level/levelgen/feature/BlockPileFeature.mapping

```diff
@@ -3,3 +3,8 @@ CLASS net/minecraft/world/level/levelgen/feature/BlockPileFeature
 		ARG 1 level
 		ARG 2 pos
 		ARG 3 random
+	METHOD tryPlaceBlock (Lnet/minecraft/world/level/LevelAccessor;Lnet/minecraft/core/BlockPos;Ljava/util/Random;Lnet/minecraft/world/level/levelgen/feature/configurations/BlockPileConfiguration;)V
+		ARG 1 level
+		ARG 2 pos
+		ARG 3 random
+		ARG 4 blockPileConfiguration
```

Comment:
Again, I don't think it's important to qualify the type of the config here. It's already in the class name, it's the config for this feature.